dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -822,8 +822,8 @@ export const p2pDataModel = [
           {
             name: "emissionIntensityOfElectricity",
             label: "Emission intensity of electricity",
-            description: "Applies especially for use of electric arc furnace (EAF) (gCO2e/kWh)",
-            unit: "",
+            description: "Applies especially for use of electric arc furnace (EAF)",
+            unit: "gCO2e/kWh",
```

Comment:
Again, Unit now no longer visible in upload page. Only acceptable if this will be resolved as part of DALA-2373.